waspc/examples/todoApp/todoApp.wasp

```diff
@@ -105,6 +105,15 @@ query getTasks {
   entities: [Task]
 }
 
+api fooBar {
+  fn: import { fooBar } from "@server/apis.js",
+  entities: [Task],
+  httpRoute: {
```

Comment:
We could use a tuple here if you like it better, so it is:
```
  httpRoute: (GET, "/foo/bar")
```

We use it for npm deps, so you can take a look at them to see how it is implemented.

But I am also ok with the dict if you like it better!

Reply:
Ah, that does look nice! I will consider swapping it to that for sure, thnx 👍🏻

Reply:
I think the only downside is it makes it harder to add more things in the future or make things optional, but I am not sure we need either of those at this point so in terms of ergonomics that may win!

Reply:
True, but it is hard to guess now. We might also just add those next to this then.
Whatever you think is best! Once we add union types, we will be able to say it can be `(GET, "foo/bar")` OR it can be a dict. So even if you got with `(,)` for now, we still have space to add more config options in the future, once we add union types. With that in mind I think it is ok to with a pair at the moment?

Reply:
Swapped syntax to the tuple format 👍🏻